zipline-loader/src/main/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -121,5 +109,9 @@ data class ZiplineModule(
     require (id !in dependsOnIds) {
       "Invalid circular dependency on self for [id=$id]"
     }
+
+    require(!filePath.startsWith("http")) {
```

Comment:
Nuke this? I think it _can_ be a relative path, but doesn’t need to be.

Also httpClient.js is a completely valid name!